server/apps/io/update_ingest_tests.py

```diff
@@ -50,9 +50,13 @@ def test_the_second_update_fails_if_already_running(self):
 
 
 class UpdateIngestTest(SuperdeskTestCase):
```

Comment:
Can we add one more test for checking if subject is empty when there's no mapping vocabulary and a mapping vocabulary with inactive entries?